tools/check-treeshaking/importChecker.js

```diff
@@ -4,15 +4,15 @@ const path = require('path');
 const forbiddenDependencies = require('./forbiddenDependencies');
 
 // thanks to https://github.com/webpack/webpack/issues/2090#issuecomment-302643018
-function findEntry(mod) {
-  if (mod.reasons.length > 0 && mod.reasons[0].module.resource) {
-    return findEntry(mod.reasons[0].module);
+function findEntry(mod, compilation) {
```

Comment:
keep these changes local if they were required for building locally to avoid extra PR reviewing overhead

Reply:
The change has been merged in another PR, should be good once I merge main again